### PR TITLE
fix visibility of testdata for //test/cmd:legacy-script

### PR DESCRIPTION
**Pull request by BenTheElder**
**What this PR does / why we need it**: fixes some BUILD visibility, albeit for a relatively unused target

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/cmd/BUILD [hunk 1]
```diff
@@ -48,6 +48,7 @@ sh_library(
         "version.sh",
```
Comment (ixdy) [inline test/cmd/BUILD]: this should maybe be a glob, too.

also the comment here doesn't really make sense.

**Comment by BenTheElder** [comment]
flake deleting firewall rules, @shyamjvs 
/test pull-kubernetes-e2e-gce-100-performance

**Pull request merged**